docker-compose.yml

```diff
@@ -9,8 +9,13 @@ services:
     container_name: mock_core
     restart: always
     ports:
-      - "4200:4200"
+      - "0.0.0.0:4200:4200"
```

Comment:
How come only the first one needs to have the host before the port?

Reply:
Oops that must have been added while I was debugging networking issues. Will remove